app/models/post.rb

```diff
@@ -2,4 +2,6 @@ class Post < ApplicationRecord
   belongs_to :user
```

Comment:
- [ ] According to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) you should have post.rb belongs to the author and it should have classname 'User'.